lib/github/cache.nit

```diff
@@ -29,7 +29,7 @@
 # var name = "privat/nit"
```

Comment:
You can add `api.clear_cache` at the beginning of the test to avoid the crash on the first assert.

My bad, sorry.